BostonData/lambda_function/lambda_function.py

```diff
@@ -116,6 +116,8 @@ def on_intent(intent_request, session):
     elif intent_name == "AMAZON.StopIntent" or \
                     intent_name == "AMAZON.CancelIntent":
         return handle_session_end_request()
+    elif intent_name == "GetAlertsIntent" :
+        return get_alerts_intent(intent, session)

```

Comment:
When I run this I get: 
`
name 'get_alerts_intent' is not defined: NameError
Traceback (most recent call last):
File "/var/task/lambda_function.py", line 46, in lambda_handler
return on_intent(event['request'], event['session'])
File "/var/task/lambda_function.py", line 120, in on_intent
return get_alerts_intent(intent, session)
NameError: name 'get_alerts_intent' is not defined
`

We probably need to import this function at the top of this file